group-coordinator/src/test/java/org/apache/kafka/coordinator/group/OffsetMetadataManagerTest.java

```diff
@@ -1491,10 +1526,15 @@ public void testConsumerGroupOffsetFetchWithUnknownMemberId() {
                 .setPartitionIndexes(Collections.singletonList(0))
         );
 
+        // Fetch offsets cases.
+        assertThrows(UnknownMemberIdException.class,
+            () -> context.fetchOffsets("group", "", 0, topics, Long.MAX_VALUE));
```

Comment:
Is this testing that the empty member string will fail (ie if no member, it must be null?) I'm trying to understand how this test is different than the one below.

I also noticed that the test changed the default values from empty string to null. Was that an error before?

Reply:
Or is this part of the latest commit that made null the no member ID indicator?

Reply:
Yeah, I changed the member id to be nullable and null by default. I think that it makes more sense. So now, an empty string raises an unknown member id error.